Containers/postgresql/start.sh

```diff
@@ -99,7 +99,7 @@ if ( [ -f "$DATADIR/PG_VERSION" ] && [ "$PG_MAJOR" != "$(cat "$DATADIR/PG_VERSIO
     fi
 
     # Get the Owner
-    DB_OWNER="$(grep -a "$GREP_STRING" "$DUMP_FILE" | head -1 | grep -oP 'Owner:.*$' | sed 's|Owner:||;s| ||g')"
+    DB_OWNER="$(grep -a "$GREP_STRING" "$DUMP_FILE" | head -1 | grep -oP 'Owner:.*$' | sed 's|Owner:||;s| ||g;s|\s||g')"
```

Comment:
`\s` matches any whitespace. therefore you can remove `;s|| |g` in the `sed` command

Reply:
also you may want to use `[[:space:]]` which is POSIX, whereas `\s` is only GNU according to [stackexchange](https://unix.stackexchange.com/a/583886/52916) (I could not find it in the manuals)

this works against my local database export:
```sh
❯ grep "Name: oc_appconfig; Type: TABLE; Schema: public; Owner:" database-dump.sql.bak | grep -oP 'Owner:.*$' | sed 's|Owner:||;s|[[:space:]]||g' | hexdump -C 
00000000  6e 63 61 64 6d 69 6e 0a                           |ncadmin.|
00000008
```

Reply:
Thanks for the idea! I just used `[[:space:]]` in the replace command. Can you please check again? :)